examples/shop-with-blog/client/src/pages/shop/Category/Category.js

```diff
@@ -38,9 +37,7 @@ const CategoryPage = ({ match: { params } }) => (
 
           return (
             <CategoryLayout variant={!filtersData.length && 'noFilters'}>
-              <Helmet>
-                <title>{name}</title>
-              </Helmet>
+              <SEO meta={category.seo} title={name} />
```

Comment:
> should not we add also fields like image or urlPath? these are also necessary from SEO perspective

(referring to this comment)

`category` (I think `Product` or `BlogPost` also) contains more SEO information like mentioned above `image`. So maybe we could introduce interface (`ISeoEntity`) which could be later implemented via `Category`, `Product`, `BlogPost`... then above line could look like:
```suggestion
              <SEO {...category} />
```

I think Interface is better than duplicate all fields into `entity.seo` node

Reply:
@tutok the proposed idea is great, but I'm afraid merging fields into the main type could lead to unwanted behavior like having a need to use a custom title for `<title>` tag (for example "Buy these cool BrandName X100 headphones"), while keeping `<h1>` more simple for visitors (for example "BrandName X100 headphones") - this way `title` is not a duplicate and serves different purposes. Same goes to `description` - `description` of the Category may contain a fully-featured HTML content, while `seo.description` is limited (by length) and must be a plaintext.

Besides that, keeping SEO-related stuff under `seo` key lets other developers using this data from another source (like external SEO-tool) while keeping the main Product/Category data to be fetched from the shop-backend.